### ISSUE TITLE
MEAN_A = np.array([113.40,114.08,116.45]) STD_A = np.array([48.30,46.27,48.14])

### ISSUE THREAD

**Issue by ranmengying** [ggsDing/Bi-SRNet#30]

**Comment by ggsDing** [comment]
> What are these based on?

calculations on the dataset.

**Comment by ranmengying** [comment]
If I want to use my own dataset, do I need to change that, and if so, how?
My own dataset image size is 256, how can I change 512 to 256 in code
In your paper, only the test set and the training set are divided, but in the code, only the two data sets are divided, and the code will give an error

**Comment by ggsDing** [comment]
> If I want to use my own dataset, do I need to change that, and if so, how? My own dataset image size is 256, how can I change 512 to 256 in code In your paper, only the test set and the training set are divided, but in the code, only the two data sets are divided, and the code will give an error

You can try modify the dataloder accordingly to load your data.
